src/common/vulkan_platform.h

```diff
@@ -59,26 +61,62 @@ class VkNonDispatchableHandle {
         return mHandle != 0;
     }
 
-    static VkNonDispatchableHandle<Tag> CreateFromHandle(uint64_t handle) {
+    // The regular Vulkan handle type depends on the pointer width but is always 64 bits wide.
```

Comment:
This reads as "The regular Vulkan handle type depends on the pointer width" + "The regular Vulkan handle type is always 64 bits wide" - sounds like a contradiction.

Reply:
Done, explained better what this means.